### ISSUE TITLE
Values in dictionnaries cannot be integer

### ISSUE THREAD

**Issue by xkobal** [KnpLabs/DictionaryBundle#89]
Hi,

Since the version 2.10, we can't use anymore integer as value of a dictionnary.

Here is an example:

```yaml
deadline:
  type: 'key_value'
    content:
      "0": deadline.now
      "15": deadline.15
      "30": deadline.30
      "60": deadline.60
```

Symfony/YAML cast "0" as an integer and throw a PHP Fatal Error:

`PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: preg_match() expects parameter 2 to be string, integer given in /app/vendor/knplabs/dictionary-bundle/src/Knp/DictionaryBundle/Dictionary/ValueTransformer/ConstantTransformer.php:24` 

It comes from the 
```php
declare(strict_types=1);
```
instruction at the beginning of the ConstantTransformer class.

Is Integer value in dictionnary configuration something forbidden now ? Or maybe you need to cast the value as string to pass the preg test.

Thanks for your help.

Xavier

**Comment by xkobal** [comment]
Thanks for the fix !

**Comment by PedroTroller** [comment]
@xkobal thanks ! 

Fixed with v2.1.1
